fuse_models/include/fuse_models/parameters/acceleration_2d_params.h

```diff
@@ -35,6 +35,7 @@
 #define FUSE_MODELS_PARAMETERS_ACCELERATION_2D_PARAMS_H
 
 #include <fuse_core/loss.h>
+#include <fuse_core/parameter.h>
```

Comment:
The changes here and in the rest of the files from here are a consequence of creating this header and moving two free functions that were used here into the `fuse_core` namespace.